bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -215,5 +215,6 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
 {% endblock %}
 
 {% block js %}
```

Comment:
Please add a brief comment here about why this was required, and note that we only need it until we get the unminified source.